h/static/styles/help-page.scss

```diff
@@ -4,6 +4,7 @@
   padding-top: 2.5em;
   padding-bottom: 2.5em;
   background: $body-background;
+  font-family: 'Lato', sans-serif !important;
```

Comment:
I think it's best if we use it just on the welcome page for now. I think the sidebar should have it's own font, since it is brand neutral. So I don't think we should use lato everywhere. 

The welcome page on the other hand should look like it's part of the Hypothes.is website, so we'll use Lato there.